lambdas/webhook_processor/webhook_processor.py

```diff
@@ -0,0 +1,163 @@
+import base64
+import json
+import os
+import urllib
+
+from botocore.vendored.requests import post
+
+import logging
+logger = logging.getLogger()
+
+# First log the function load message, then change
+# the level to be configured via environment variable.
+logger.setLevel(logging.INFO)
+logger.info('Loading function')
+
+# Log level is set as a string, default to 'INFO'
+log_level = os.environ.get('LOG_LEVEL', 'INFO').upper()
+numeric_level = getattr(logging, log_level, None)
+if not isinstance(numeric_level, int):
+    raise ValueError('Invalid log level: {}'.format(log_level))
+logger.setLevel(numeric_level)
+
+
+def _get_target_urls():
+    """
+    Get the target URLs for the processed hooks from a comma delimited list of URLs,
+    set in an OS environment variable.
+
+    Return a list of target URLs
+    """
+    url = os.environ.get('TARGET_URLS')
+    if not url:
+        raise StandardError('Environment variable TARGET_URLS was not set')
+
+    return url.split(',')
```

Comment:
Are you intending this variable to represent multiple URLS, each of which will receive the same payload? Each Jenkins instance has separate endpoints for GHRPB requests, merges and buildWithParameters calls